packages/design-system-twrnc-preset/jest.config.js

```diff
@@ -14,13 +14,17 @@ module.exports = merge(baseConfig, {
   // The display name when running multiple projects
   displayName,
 
+  // TODO add tests to twrnc preset https://github.com/MetaMask/metamask-design-system/issues/90
+  // Pass with no tests if no test files are found
+  passWithNoTests: true,
```

Comment:
We still don't have any tests for the twrnc preset package have commented the issue https://github.com/MetaMask/metamask-design-system/issues/90